### PR TITLE
#155 Update reference sequence numbering diagram

### PR DESCRIPTION
**Pull request by reece**
See #155

### CODE REVIEW

**Comment by reece** [comment]
Here's the sole change:
https://hgvs-nomenclature--156.org.readthedocs.build/156/background/numbering/#figure

Compare with
https://hgvs-nomenclature.org/background/numbering/#figure

File: docs/stylesheets/extra.css [hunk 1]
```diff
@@ -145,7 +145,7 @@ code::first-of-type and code::last-of-type should get left and right padding res
 
 figure {
   border: thin solid var(--md-typeset-a-color);
-  background-color: #eee;
```
Comment (ahwagner) [inline docs/stylesheets/extra.css:148]: I happened to look at this figure this evening, while my browser was in dark mode. It looks like this:

image

I propose keeping the background styling for figures white–or at least doing so for this figure.

**Review by ahwagner — CHANGES_REQUESTED** [review]

**Comment by reece** [comment]
Good catch @ahwagner !

**Review by ifokkema — APPROVED** [review]
Looks good to me!

**Review by ahwagner — APPROVED** [review]

**Review by jfjlaros — APPROVED** [review]
This looks fine to me.

**Pull request merged**